actors/builtin/miner/miner_actor.go

```diff
@@ -290,10 +288,11 @@ func (a Actor) PreCommitSector(rt Runtime, params *PreCommitSectorParams) *adt.E
 
 	// Request deferred Cron check for PreCommit expiry check.
 	cronPayload := CronEventPayload{
-		EventType: CronEventType_Miner_PreCommitExpiry,
-		Sectors:   []abi.SectorNumber{params.Info.SectorNumber},
+		EventType:       CronEventType_Miner_PreCommitExpiry,
+		Sectors:         []abi.SectorNumber{params.Info.SectorNumber},
+		RegisteredProof: params.RegisteredProof,
 	}
-	expiryBound := rt.CurrEpoch() + PoRepMaxDelay + 1
+	expiryBound := rt.CurrEpoch() + MaxSealDuration[params.RegisteredProof] + 1
```

Comment:
can pull the `regProof` from `MinerInfo` in actor state + invalid if not verified as is

Reply:
It needs to be per sector actually, so submitted with `PreCommitInfo`